MMM-MBTA.js

```diff
@@ -436,9 +452,9 @@ Module.register("MMM-MBTA", {
         // Gets (maxEntries + 10) schedules or all schedules up to 3 hours from now, whichever is lower
         // Page and time limits necessary because otherwise "schedules" endpoint gets every single schedule
         if (!this.config.predictedTimes) {
-            url += "&page[limit]=" + (maxEntries + 10) + '"';
+            url += "&page[limit]=" + (this.config.maxEntries + 10) + '"';
             url += "&filter[min_time]=" + moment().format("HH:mm");
-            url += "&filter[max_time]=" + moment().add(3, 'h').format("HH:mm");
+            url += "&filter[max_time]=" + moment().add(5, 'h').format("HH:mm");
```

Comment:
Any particular reason for this change? if you want to keep it at five, make sure to update the comment on line 452!

Reply:
Alright, that's understandable! Just update the comment and you should be all set for this part!